views/js/pciCreator/helper/responseCondition.js

```diff
@@ -24,13 +24,15 @@ define(['lodash', 'jquery', 'taoQtiItem/qtiCreator/helper/xmlRenderer'], functio
         var rp = item.responseProcessing;
         var renderedRp = rp.render(xmlRenderer.get()) || '<responseProcessing template=\"EMPTY\"/>';
         var $rpXml = $($.parseXML(renderedRp));
-        var newRc = $rpXml[0].importNode($.parseXML(newResponseConditionXml).documentElement, true);
+        var rcXml = $.parseXML(newResponseConditionXml);
+        var newRc = $rpXml[0].importNode(rcXml.documentElement, true);
         var responseIdentifier = interaction.attr('responseIdentifier');
         var responseDeclaration = interaction.getResponseDeclaration();
+        var $respVar = $(rcXml).find('variable[identifier="'+responseIdentifier+'"]');
 
         //prepare replacement criteria
         criteria = _.defaults(criteria || {}, {
-            responseIdentifierCount : 1
+            responseIdentifierCount : $respVar.length
```

Comment:
Instead of this I could count all RESPONSE from template, of course if this has sense for you